app/jobs/comments/bust_cache_job.rb

```diff
@@ -7,6 +7,8 @@ def perform(comment_id, service = EdgeCache::Commentable::Bust)
       return unless comment&.commentable
 
       comment.purge
+      comment.commentable.purge
```

Comment:
This should help us transition to the `fastly-rails` based system of purging in the future. I'm not 100% sure it will purge correctly but I can only test purging in production :)

Reply:
Since we allow comments to become orphaned from the commentable so I think this needs safe navigation.

Reply:
I didn't add it because on line 7 this job would quite if commentable is not available. I've improved the tests to make sure that it doesn't break in https://github.com/thepracticaldev/dev.to/pull/5543/commits/3c96c50db8c549de78fcc2c663451ed731cb28e2

Reply:
Oh right. Good call.